src/objects/npc.lua

```diff
@@ -30,14 +30,56 @@ return {
         self.is_walking = false
         self.direction = 'right'
 
+        self.colors = {
```

Comment:
The colors list here is a map for NPC names which is read on line 43 -- perhaps we can have separate fields for the NPC color and the NPC name in the map?